src/main/java/com/softserve/academy/spaced/repetition/logger/LogException.java

```diff
@@ -1,27 +1,27 @@
 package com.softserve.academy.spaced.repetition.logger;
 
+import com.softserve.academy.spaced.repetition.Application;
 import org.aspectj.lang.JoinPoint;
 import org.aspectj.lang.annotation.AfterThrowing;
 import org.aspectj.lang.annotation.Aspect;
-import org.springframework.beans.factory.annotation.Autowired;
+import org.slf4j.LoggerFactory;
 import org.springframework.stereotype.Component;
+import org.slf4j.Logger;
 
 @Aspect
 @Component
 public class LogException {
-    @Autowired
-    Logger logger;
+
+    private static final Logger logger = LoggerFactory.getLogger(Application.class);
```

Comment:
The parameter value should be the class of this object.

Reply:
See com.softserve.academy.spaced.repetition.controller.CardController for example.

Reply:
Oh, sorry. I'll fix it.